presto-hive/src/main/java/com/facebook/presto/hive/parquet/ParquetTypeUtils.java

```diff
@@ -125,12 +186,21 @@ public static ParquetEncoding getParquetEncoding(Encoding encoding)
         }
         // parquet is case-sensitive, but hive is not. all hive columns get converted to lowercase
         // check for direct match above but if no match found, try case-insensitive match
-        for (Type type : messageType.getFields()) {
+        for (parquet.schema.Type type : messageType.getFields()) {
             if (type.getName().equalsIgnoreCase(columnName)) {
                 return type;
             }
         }
 
         return null;
     }
+
```

Comment:
now that we have this method in `ParquetTypeUtils` we can update `ParquetColumnReader` to also call this one instead of maintaining two copies.